components/bullet/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.illumos.org/license/CDDL.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2015, Aurelien Larcher. All rights reserved.
+#
+include ../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		bullet
+COMPONENT_VERSION=	2.83.5
+COMPONENT_FMRI=     	library/bullet
+COMPONENT_CLASSIFICATION=System/Libraries
+COMPONENT_SRC=		bullet3-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	http://bulletphysics.org/
+COMPONENT_ARCHIVE=	$(COMPONENT_VERSION).tar.gz
```

Comment:
OK, I did not realize that the tarball would be renamed to $(COMPONENT_ARCHIVE), I should add some note to make it clear on the wiki.